README.md

```diff
@@ -70,3 +70,11 @@ Press CTRL+C to quit
  * Debugger PIN: 113-666-066
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+
+## Testing
+
+Pytest is used to test functionality
+
+To run all tests , run python3 -m pytest
+To run individual tests, run python3 -m pytests <testfile>
```

Comment:
I'd recommend ensuring that your commands use `poetry` explicitly in order to ensure you're using the right virtual environment, e.g.:
```suggestion
To run all tests , run `poetry run python3 -m pytest`
To run individual tests, run `poetry run python3 -m pytests <testfile>`
```